### DISCUSSION TITLE
Profile for IDEX

### DISCUSSION THREAD

**Discussion by hliebscher** [OrcaSlicer/OrcaSlicer#303] [Q&A] [answered]
i am looking for a working IDEX setting or printer Profile with dual extruder

**Comment by hliebscher** [comment, marked as answer]
now i have self made ;-)

**Reply by mrhasbean** [reply]
Are you willing to share or at least explain how to create it? I would love a profile for my Sovol SV04 which is on Klipper. Cheers.

**Comment by hliebscher** [comment]
@mrhasbean look at the start and end gcode and the code for each filament (left and right)
you have to make the g-code for the both printheads in filament c-code
 
[idex.3mf.zip](https://github.com/SoftFever/BambuStudio-SoftFever/files/10834331/idex.3mf.zip)

**Reply by chasedown26** [reply]
> @mrhasbean look at the start and end gcode and the code for each filament (left and right) you have to make the g-code for the both printheads in filament c-code
> 
> [idex.3mf.zip](https://github.com/SoftFever/BambuStudio-SoftFever/files/10834331/idex.3mf.zip)

So i have Gcode from prusaslicer for tool change where would this go?

**Comment by hliebscher** [comment]
@mrhasbean do you need any informations? or  is fixed?

**Reply by cryoglobe** [reply]
Yes ... would be nice to see the "Sovol Sv04 Idex "  ( not klipper ) printer in the list :)

**Comment by B20bob** [comment]
Huge thanks to @hliebscher I was able to work up machine and filament profiles to get Orca working on the SV04 running the stock Marlin FW:

https://github.com/csn495/Orcaslicer_Sovol_SV04_DUAL_Profiles
